src/scancode/plugin_ignore_copyrights.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (c) 2018 nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+
+from plugincode.output_filter import OutputFilterPlugin
+from plugincode.output_filter import output_filter_impl
+from scancode import CommandLineOption
+from scancode import OUTPUT_FILTER_GROUP
+
+
+@output_filter_impl
+class IgnoreCopyrights(OutputFilterPlugin):
+    """
+    Filter findings that match given copyright holder or author patterns.
+    Has no effect unless the --copyright scan is requested.
+    """
+
+    options = [
+        CommandLineOption(('--ignore-copyright-holders',),
```

Comment:
I wonder if this should be singular as in `ignore-copyright-holder` and `ignore-author` since each option in the CLI accepts only one value, and multiple ones are allowed?

Reply:
Good point. I went with plural since I was thinking of it as the "ignore copyrights" plugin. But I agree that singular is better. I'll change it.